tests/pipe/pure-filer.pipe.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { PureFilterPipe } from '../../src/pipe/pure-filter.pipe';
+
+
+describe('PureFilterPipe Tests', () => {
+    let pipe: PureFilterPipe;
+    let list: Array<any>;
+    let filter;
+    beforeEach(() => {
+        pipe = new PureFilterPipe();
+    });
+
+    it('Should filter by  string', () => {
+        list = ['java', 'javaScript', 'TypeScript'];
+        const result = pipe.transform(list, 'java', null);
+        expect(result).toEqual(['java', 'javaScript']);
+        });
```

Comment:
Right click and you will see format document. You could also install the beautify plugin if you want